api/src/utils/index.ts

```diff
@@ -19,3 +20,12 @@ function sha256(buf: Buffer) {
   return createHash('sha256').update(buf).digest();
 }
 export const challenge = base64URLEncode(sha256(Buffer.from(verifier)));
+
+/**
+ *
+ * @param obj This is the full challenge object.
+ * @returns This is the challenge object with only the id and completedDate.
+ */
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export const fixPartiallyCompletedChallengeItem = (obj: any) =>
```

Comment:
This is a nit, so don't mind me, but we should be able to do something like:

`obj: { id: string, completedDate: number, [key: string]: unknown}`

This will at least force us to validate those properties exist when passing in an argument to this function.

Reply:
I don't think we even need this function. `partially CompletedChallenges` is defined in the schema, so we should be able to use it un-fixed.

Reply:
so what is the best option?

Reply:
It's effectively doing `x => x`, so you can delete it.